src/index.js

```diff
@@ -5,23 +5,20 @@ import 'angular-translate';
 import 'angular-leaflet-directive';
 import 'ovh-api-services';
 
-import { PICTURES, MONDIAL_RELAY } from './constants';
-
-import directive from './directive';
+import component from './component';
 
 import './index.less';
 
 const moduleName = 'ngOvhMondialRelay';
+const componentName = 'ngOvhMondialRelay';
 
 angular
   .module(moduleName, [
     'leaflet-directive',
     'ovh-api-services',
     'pascalprecht.translate',
   ])
-  .constant('MONDIAL_RELAY', MONDIAL_RELAY)
-  .constant('MONDIAL_RELAY_PICS', PICTURES)
-  .directive('mondialRelay', directive)
+  .component(componentName, component)
```

Comment:
Keeping `mondialRelay` as component name is fine. `ng` prefixes are use to define angular module.

Is that make sense for you?

Or it also could be named as `ovhMondialRelay` because some services are used such as `OvhApiMe` and `OvhApiSupplyMondialRelay`.

But bear in mind that in both cases we are introducing a 💥 Breaking Change.

Reply:
Ok I'll change the name to `ovhMondialRelay` to keep consistency with providers prefixed by `ovh` so